index.html

```diff
@@ -199,41 +200,41 @@ <h1>Built for Ethereum</h1>
     </div><!-- //Container -->
   </section><!-- //Status -->
 
-  <section id="research">
+  <section id="install">
     <div class="container">
       <div class="row">
         <div class="col-xs-12 col-md-12 text-left">
           <div class="page-header">
-            <h1>Research</h1>
+            <h1>Installing Swarm</h1>
           </div>
-          <p class="lead">
-            The ethersphere orange paper series is an attempt to provide an umbrella for sharing and publishing technical reports of the ongoing research on formal foundations of web3 base layer infrastructure. Our aim is to foster synergy and collaboration between various groups and individuals working in the ethersphere.
-            The Orange of the orange paper pays hommage to the yellow paper but tainted with the blood sweat and tears of cutting edge research.
+          <h3>Install on Ubuntu via PPAs</h3>
+
+          <p class="lead">The simplest way to install Swarm on Ubuntu distributions is via the built in launchpad PPAs (Personal Package Archives). We provide a single PPA repository that contains our stable releases for Ubuntu versions trusty, xenial, artful, bionic and cosmic.</p>
```

Comment:
since this is only for Ubuntu, shouldn't we add a section for everyone else: "the simplest way is to download Geth&all-tools and copy the binaries."
...or are you thinking we'll remove swarm from that package anyway?

Reply:
@homotopycolimit in the PR I submitted that renames `swarm` to `ethereum-swarm`, I am removing `swarm` from `geth-alltools` archive. However I still keep `ethereum-swarm` as a dependency of `ethereum`, so that users get swarm, when they run `apt install ethereum`.

Do you mean we should also explain how to install Geth here? I am not sure I follow where the Geth&AllTools fits here.

Reply:
if you go to geth.ethereum.org and download "geth&alltools", then you will have a swarm binary that you can use. That's all I meant.

Reply:
@homotopycolimit oh I see what you mean.

This PR is mostly about fixing it properly, once we address the versioning and deb naming problem. It wasn't intended to fix the wrong docs for the meantime.

We could probably issue a fix where we tell people to go to geth.ethereum.org for the next 1-2 weeks, but I am not sure if it is worth doing it right now.